deployment/provider-azure-installer.yaml

```diff
@@ -15,7 +15,7 @@ spec:
       tolerations:
       containers:
         - name: provider-azure-installer
-          image: aramase/secrets-store-csi-driver-provider-azure:v0.0.6
```

Comment:
Do we want to point this at an image in a organization's repo instead of a user's repo?